cinema/views.py

```diff
@@ -56,3 +92,60 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
+
+        if self.action == "list":
+            queryset = (
+                queryset.
+                select_related("movie", "cinema_hall").
+                annotate(tickets_available=F(
+                    "cinema_hall__rows") * F(
+                    "cinema_hall__seats_in_row") - Count("tickets")
+                )
+
+            )
```

Comment:
this is also a bad formatting
closing/opening bracket both should end/start with new line OR at the same line as data